packages/driver/src/cross-origin/origin_fn.ts

```diff
@@ -66,6 +61,38 @@ const rehydrateRunnable = (data: serializedRunnable): Runnable|Test => {
   return runnable
 }
 
+const getCallbackFn = async (fn: string, file?: string) => {
+  if (
+    // @ts-ignore
```

Comment:
```suggestion
    // @ts-expect-error
```

Reply:
I wonder if it makes sense to move the `experimentalOriginDependencies` flag into `CoreConfigOptions` and let the `config` package run enforcement of the e2e config in order to get the typings support. Though I can't see users referencing this

Reply:
We should possibly do something like that. It's awkward that the types aren't quite right for testing type-specific config values. I just didn't want to mess with that in the scope of this PR.